pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,102 @@
+---
+$title: Use AMP components in non-AMP pages
+$order: 1
+description: 'Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP.'
+tutorial: false
+toc: true
+formats:
+  - websites
+author: CrystalOnScript
+---
+
+Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP. Bento AMP is a project that embraces use of AMP components without the need to commit to fully valid AMP. This creates copy and past-able code for common UI widgets, allows testing if AMP is the right switch for your website, and using Bento in a migration plan. 
+
+[tip type="important"]
+ Bento is only available [experimentally](../learn/experimental.md). Read our [Blog Post](TODO) for information on future plans and how to get involved! 
+[/tip]
+
+# Use Bento components in non-AMP pages
+
+Bento components are available for experimental use in the developer preview. After enabling the Bento experimental 
+
+
+## Enable Bento Experiment 
+
+Bento components are available experimentally and you must [enable experimental features](../learn/experimental.md).
+
+You can enable all Bento components by including the script below:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento', true);
+  });
+</script>
+```
+
+
+Or enable a single Bento component by including the script below. Ensure you replace `bento-foo` with the desired supported component name given on its documentation page:
```

Comment:
would it make sense to link "supported component" to the "Available Bento components" section?

Reply:
good suggestion - done!